src/sentry/utils/retries.py

```diff
@@ -76,6 +77,8 @@ def __call__(self, function):
                 try:
                     return function()
                 except self.exceptions as error:
+                    if self.log_original_error:
+                        logger.info(error)
```

Comment:
Should this be `info` or a different severity level (I'm assuming there are others)

Reply:
@billyvg Good question. When you use `debug`, it doesn't even show up in my machine. I am guessing that the default log level is only at info. I figured since we weren't logging this error before the safest thing to do is make it the lowest level that will actually help people, which is `info`.